packages/react/src/KeybindingHint/components/Chord.tsx

```diff
@@ -31,13 +31,10 @@ const splitChord = (chord: string) =>
     .map(k => k.toLowerCase())
     .sort(compareLowercaseKeys)
 
-export const Chord = ({keys, format = 'condensed', variant = 'normal', size = 'normal'}: KeybindingHintProps) => (
+export const Chord = ({keys, format = 'condensed', size = 'normal'}: KeybindingHintProps) => (
```

Comment:
Does this mean the "variant" prop in [KeyBindingHint](https://github.com/primer/react/blob/ececdb914eb0d0c5bd9699681386da744a470a76/packages/react/src/KeybindingHint/props.ts#L29) will be a no-op? Not sure we should remove this without a major 🤔